test/src/cdp/pdf.spec.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {readFile, unlink} from 'fs/promises';
+
+import expect from 'expect';
+
+import {getTestState, setupTestBrowserHooks} from '../mocha-utils.js';
+
+describe.only('Page.pdf', () => {
+  setupTestBrowserHooks();
+
+  it('can print to PDF with accessible', async () => {
```

Comment:
this will conflict with the other PR

Reply:
Let's merge it and I will resolve the conflicts on the other PR, I think we should flip this as well.

Reply:
it is very new so perhaps we won't turn it on by default